code/modules/power/tesla/energy_ball.dm

```diff
@@ -91,8 +92,21 @@
 /obj/singularity/energy_ball/proc/move_the_basket_ball(var/move_amount)
 	//we face the last thing we zapped, so this lets us favor that direction a bit
 	var/move_bias = pick(GLOB.alldirs)
+	var/move_dir
+	var/rods = GLOB.grounding_rods
+	if(rods) // grounding rods pull the tesla ball, picks the nearest one
```

Comment:
this is unnecessary, if GLOB doesn't have grounding_rods it'll yell anyway afaik

Reply:
i did this for readability more than anything, would removing the if(rods) and replacing the for(in rods) with (in GLOB) work?